crypto/err/err.c

```diff
@@ -839,17 +721,7 @@ const char *ERR_reason_error_string(unsigned long e)
 
 void ERR_remove_thread_state(const CRYPTO_THREADID *id)
```

Comment:
Right. Though we still need a "remove current thread state", since it's used by the init code, should I add a new function?